### PR TITLE
test: add support for test video capture and rendering to Kalium CLI (WBP-2101)

### PR DESCRIPTION
**Pull request by c-g-owen**
* Adds support for video capture and rendering with real capturer-renderers (OSX) and fake capturer-renderers (OSX & Linux) in Kalium-cli.
* Updates AVS dependency to 9.7.1 to make use of new kcall API providing above.

#### How to Test

Run Kalium CLI with "console" option, make call, use V command to toggle outgoing video. Incoming video is shown for real renderers automatically.
Add "-T" option to enable testing mode (auto-generated video capture and dummy renderers).

### CODE REVIEW

**Comment by c-g-owen** [comment]
Please note that the update to AVS 9.7.1 is necessary for this PR, as there are new AVS functions used in this code (namely the kcall_ functions) that are only present in AVS 9.7.1 and above

File: cli/src/jvmMain/kotlin/com/wire/kalium/cli/commands/ConsoleCommand.kt [hunk 1]
```diff
@@ -62,9 +66,24 @@ class ConsoleCommand : CliktCommand(name = "console") {
         KeyStroke('e', ::endCallHandler),
         KeyStroke('m', ::muteCallHandler),
         KeyStroke('s', ::selectConversationHandler),
+        KeyStroke('V', ::toggleVideoHandler),
```
Comment (typfel) [inline cli/src/jvmMain/kotlin/com/wire/kalium/cli/commands/ConsoleCommand.kt:69]: @c-g-owen should it be capital `V`?

**Review by alexandreferris — COMMENTED** [review]
Great additions!

I just have some nitpick comments.

Also it is missing iOS implementation on `CallManagerImpl.kt`

Another thing, if we can add tests for `SetTestPreviewActiveUseCase`/`SetTestRemoteVideoStatusUseCase`/`SetTestVideoTypeUseCase` even though it is for tests, it would be great to keep the tests tested 😂

**Review by ohassine — APPROVED** [review]

**Comment by typfel** [comment]
> Great additions!
> 
> I just have some nitpick comments.
> 
> Also it is missing iOS implementation on `CallManagerImpl.kt`
> 
> Another thing, if we can add tests for `SetTestPreviewActiveUseCase`/`SetTestRemoteVideoStatusUseCase`/`SetTestVideoTypeUseCase` even though it is for tests, it would be great to keep the tests tested 😂

@alexandreferris added the missing tests

**Review by alexandreferris — APPROVED** [review]

**Comment by typfel** [comment]
@alexandreferris any idea why the instrumented tests started to fail? Are they broken on develop?

**Pull request merged**